### DISCUSSION TITLE
Unable to signup

### DISCUSSION THREAD

**Discussion by v00ts** [vercel/community#5600] [Account Issue] [answered]
### Question

Hi there,
I've been trying to sign up with my GitHub account and it has been telling me the same reply: There is already an account associated with your GitHub email address. Please login with your email and password then link your GitHub account.
I tried login with my email and I got this response: we are unable to complete your signup.

**Comment by swarnava** [comment, marked as answer]
Please email us at registration@vercel.com using your registered email address.
